server/channels/app/import_functions.go

```diff
@@ -1272,10 +1273,60 @@ func (a *App) importReplies(rctx request.CTX, data []imports.ReplyImportData, po
 	return nil
 }
 
+func compareFilesContent(fileA, fileB io.Reader, bufSize int64) (bool, error) {
+	aHash := sha1.New()
+	bHash := sha1.New()
+
+	if bufSize == 0 {
+		bufSize = 1024 * 1024 * 2 // 2MB
```

Comment:
Can we perhaps add a comment to this PR mentioning that after some tests we found that 2MB is the sweet spot for buffer allocation?

Reply:
Yeah, was waiting to have the PR link ready.